frontend/src/core/user/components/UserMenu.js

```diff
@@ -88,7 +88,7 @@ export class UserMenuBase extends React.Component<Props, State> {
                 { !user.isAuthenticated ? null :
                 <React.Fragment>
                     <li className="details">
-                        <a href="/profile/">
+                        <a href={ `/contributors/${user.username}/` }/>
```

Comment:
Please remove the trailing slash and we're good to go!

Reply:
Done :)

Reply:
Sorry, I removed the wrong one.  Resending now